docs/src/config/boundaries.md

```diff
@@ -162,6 +162,35 @@ with
 `"Attributes" [None]` :  Integer array of mesh boundary attributes at which to apply the
 PMC boundary condition.
 
+## `boundaries["Impedance"]`
+
+```json
+"Impedance":
+[
+    {
+        "Attributes": [<int array>],
+        "Rs": <float>,
+        "Ls": <float>,
+        "Cs": <float>
+    },
+    ...
+]
+```
+
+with
+
+`"Attributes" [None]` :  Integer array of mesh boundary attributes for this surface
+impedance boundary.
+
+`"Rs" [0.0]` :  Surface resistance used for computing this surface impedance boundary's
+impedance per square, ``\Omega``/sq.
```

Comment:
The square unit here, is it meters or is it non dimensional area? Given `Omega`, `H` and `F` still have units, having the semi-nondimensional is a little confusing. If it's mesh length units, maybe specify as `/ L^2`

Reply:
This is nondimensional "per square": https://en.wikipedia.org/wiki/Sheet_resistance.